indico/modules/events/editing/clone.py

```diff
@@ -26,8 +26,12 @@ class EditingSettingsCloner(EventCloner):
     def is_visible(self):
         return self.old_event.type_ == EventType.conference
 
+    def has_conflicts(self, target_event):
+        return (bool(target_event.editing_file_types) or bool(target_event.editing_review_conditions)
```

Comment:
I just noticed that it doesn't seem to be possible to clone the event editing settings. I created a brand new event, but I can't import them because `bool(target_event.editing_file_types)` will always be Truthy, since it needs to have at least one publishable file-type, right? Perhaps we could disable this all-together, overwrite or ignore these cases?

Reply:
Should we just disable the importer for editing? Checking if there are only these three file types, unmodified, feels a bit much for this...

Reply:
`bool(target_event.editing_file_types)` is False when editing is disabled in the target event, so import is possible only then. But, like @ThiefMaster said, checking the default file types seems like too much and overall importing editing settings is not so important so I can disable it.